chains/evm/contracts/scripts/CCIPManualExecutionScript.s.sol

```diff
@@ -0,0 +1,62 @@
+pragma solidity ^0.8.24;
+
+import {Internal} from "../libraries/Internal.sol";
+import {OffRamp} from "../offRamp/OffRamp.sol";
+
+import {IERC20} from "@chainlink/vendor/openzeppelin-solidity/v5.0.2/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@chainlink/vendor/openzeppelin-solidity/v5.0.2/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import {Script} from "forge-std/Script.sol";
+
+// solhint-disable-next-line no-console
+import {console2 as console} from "forge-std/console2.sol";
+
+/* solhint-disable no-console */
+contract CCIPSendTestScript is Script {
+  using SafeERC20 for IERC20;
+
+  error ManualExecutionFailed();
+  error ManualExecutionNotAllowed();
+
+  // Ex: "ETHEREUM_RPC_URL" as defined in .env
+  string public RPC_IDENTIFIER;
+
+  OffRamp public s_offRamp;
+
+  bytes32 public s_messageId;
+
+  uint64 public s_sourceChainSelector;
+  uint64 public s_sequenceNumber;
+  bytes public s_manualExecutionData;
+
+  function run() public {
+    vm.createSelectFork(RPC_IDENTIFIER);
+
+    uint256 privateKey = vm.envUint("PRIVATE_KEY");
+
+    address sender = vm.rememberKey(privateKey);
+
+    vm.startBroadcast(privateKey);
```

Comment:
This only has to wrap the non-view calls right? So probably best to only have it cover the call